packages/@aws-cdk/aws-apigatewayv2-integrations/README.md

```diff
@@ -149,6 +150,56 @@ const httpEndpoint = new HttpApi(stack, 'HttpProxyPrivateApi', {
 });
 ```
 
+### Request parameter mapping
+
+### Request Parameters
+
+Request parameter mapping is supported through a set of helper methods.
+The following example renames a request header from header1 to header2 by generating mappings
+`append:header.header2: $request.header.header1` and `remove:header.header1: ''`:
+
+```ts
+const vpc = new ec2.Vpc(stack, 'VPC');
+const lb = new elbv2.ALB(stack, 'lb', { vpc });
+const listener = lb.addListener('listener', { port: 80 });
+listener.addTargets('target', {
+  port: 80,
+});
+
+const httpEndpoint = new HttpApi(stack, 'HttpProxyPrivateApi', {
+  defaultIntegration: new HttpAlbIntegration({
+    listener,
+    requestParameters: new RequestParameters()
+      .addParameter({
+        mappingKey: HttpMappingKey.appendHeader('header2'),
+        mappingValue: MappingValue.requestHeader('header1'),
+      })
+      .addParameter({
+        mappingKey: HttpMappingKey.removeHeader(),
```

Comment:
I suppose you meant to say `HttpMappingKey.removeHeader('header1')` ?

Reply:
Yes! I will fix that!